tests/scale_test/utils/test_utils.py

```diff
@@ -0,0 +1,136 @@
+import requests
+import requests_mock
+import pytest
+import os
+import time
+from . import utils
+import tempfile
+
+
+curr_path = os.path.abspath(os.path.dirname(__file__))
+
+
+def test_compose_label_returns_expected_dict_for_valid_default_string():
+    default_string = '{' + str('"comment": "scaling-test-{}"'.format(time.strftime('%Y-%m-%d'))) + '}'
+    assert utils.compose_label(default_string) == {
+        "comment": "scaling-test-{}".format(time.strftime('%Y-%m-%d'))
+    }
+
+
+def test_compose_label_returns_none_for_invalid_string():
+    assert utils.compose_label('random_string') is None
+
+
+def test_compose_label_returns_none_for_non_string():
+    assert utils.compose_label(None) is None
+
+
+def test_load_es_query_returns_valid_dict():
+    assert utils.load_es_query(curr_path + '/test_data/smartseq2-query.json').get('query').get('bool').get('must')[0].get('match').get('files.process_json.processes.content.library_construction_approach') == 'Smart-seq2'
```

Comment:
Optional style suggestion: you could use bracket notation here instead of chaining `.get` since this is expecting all of the keys to be present and should fail otherwise